anomalib/models/fastflow/loss.py

```diff
@@ -0,0 +1,28 @@
+"""Loss function for the FastFlow Model Implementation."""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import List
+
+import torch
+from torch import Tensor, nn
+
+
+class FastflowLoss(nn.Module):
+    """FastFlow Loss."""
+
+    def forward(self, hidden_variables: List[Tensor], jacobians: List[Tensor]) -> Tensor:
+        """Calculate the Fastflow loss.
+
+        Args:
+            hidden_variables (List[Tensor]): Hidden variables from the fastflow model. f: X -> Z
+            jacobians (List[Tensor]): Log of the jacobian determinants from the fastflow model.
+
+        Returns:
+            Tensor: _description_
```

Comment:
This is a leftover from the previous PR, but would be good to add a description here in this PR.